torch/_dynamo/eval_frame.py

```diff
@@ -99,6 +99,16 @@ class Unset(Enum):
 }
 
 
+from collections import namedtuple
```

Comment:
imports at top of file

Reply:
Fixed in [0be4618](https://github.com/pytorch/pytorch/pull/106673/commits/0be4618dad04eef01833dc26c4ff5046414e74df).